### PR TITLE
WEBUI-488: update Safari to v15 in Saucelabs

### PR DESCRIPTION
**Pull request by richardsd**
Notes: 
- Updated the MacOSx version to Big Sur (once Saucelabs update the Bigsur's Safari version, we will use it)

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -38,8 +38,8 @@ if (process.env.SAUCE_USERNAME && process.env.SAUCE_ACCESS_KEY) {
     sl_latest_safari: {
       base: 'SauceLabs',
       browserName: 'safari',
-      platform: 'macOS 10.13',
-      version: 'latest',
+      platform: 'macOS 11.00',
+      version: 'latest', // TODO - check if v15 is available
```
Comment (Gabez0r) [inline karma.conf.js]: If we merge it like this, we would eventually just get v15 once it is available, right?

Comment (richardsd) [reply karma.conf.js]: For sure. We just ned to update the platform version, and once they update the platform on their side, we'll get the v15 seamlessly.

Comment (richardsd) [reply karma.conf.js]: I propose to remove the comment and close the ticket merging the PR, wdyt?

Comment (Gabez0r) [reply karma.conf.js]: I agree!

Comment (richardsd) [reply karma.conf.js]: Proceeding with that, thanks.

**Review by Gabez0r — APPROVED** [review]

**Review by mnixo — APPROVED** [review]

**Pull request merged**